orchagent/tunneldecaporch.cpp

```diff
@@ -72,18 +89,14 @@ void TunnelDecapOrch::doTask(Consumer& consumer)
                 {
                     try
                     {
-                        ip_addresses = IpAddresses(fvValue(i));
+                        dst_ip_addresses = IpAddresses(fvValue(i));
                     }
                     catch (const std::invalid_argument &e)
                     {
                         SWSS_LOG_ERROR("%s", e.what());
                         valid = false;
                         break;
                     }
-                    if (exists)
```

Comment:
Please keep the logic as before

Reply:
The logic has to be changed as we have to wait `src_ip` filed be parsed. That why I move `setIpAttribute` after thr for loop.